flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/TestingSlotPoolImpl.java

```diff
@@ -83,4 +93,52 @@ boolean isBatchSlotRequestTimeoutCheckEnabled() {
 	public ResourceProfile getLastRequestedSlotResourceProfile() {
 		return lastRequestedSlotResourceProfile;
 	}
+
+	public void setReleaseSlotConsumer(Consumer<SlotRequestId> releaseSlotConsumer) {
+		this.releaseSlotConsumer = Preconditions.checkNotNull(releaseSlotConsumer);
+	}
+
+	public void setTimeoutPendingSlotRequestConsumer(Consumer<SlotRequestId> timeoutPendingSlotRequestConsumer) {
+		this.timeoutPendingSlotRequestConsumer = Preconditions.checkNotNull(timeoutPendingSlotRequestConsumer);
+	}
+
+	@Override
+	public void releaseSlot(
+		@Nonnull SlotRequestId slotRequestId,
```

Comment:
`@Nonnull` is not needed since it is the default assumption.
Only `@Nullable` params need to be explicitly tagged.